deploy/rpm/gedkeeper.spec

```diff
@@ -33,11 +30,6 @@ AutoReqProv:	no
 
 %prep
 %setup -qc
-find . -type f -iname "*.dll" -exec chmod -x {} \;
-find ./locales -type f -exec chmod -x '{}' \;
-find ./plugins -type f -exec chmod -x '{}' \;
-find ./scripts -type f -exec chmod -x '{}' \;
-find ./samples -type f -exec chmod -x '{}' \;
```

Comment:
Are you sure it's not needed?

Reply:
Yes, %setup macro does:
```
chmod -Rf a+rX,u+w,g-w,o-w .
```
If we want to make sure all files no executable we can add:
```
chmod -Rf a-x .
```